website/src/pages/downloads/windows.tsx

```diff
@@ -76,28 +116,71 @@ export function WindowsDownloads(): JSX.Element {
           <div className="pt-8">
             <Link
               className="mt-auto no-underline hover:no-underline inline-flex text-white hover:text-white bg-purple-500 border-0 py-2 px-6 focus:outline-none hover:bg-purple-500 rounded text-md font-semibold"
-              to={downloadData.setup}>
+              to={downloadData.setupX64}>
               <FontAwesomeIcon size="1x" icon={faDownload} className="mr-2" />
               Download Now
             </Link>
             <caption className="block w-full mt-1 text/50 dark:text-white/50">
-              Windows installer, version {downloadData.version}
+              Windows installer x64, version {downloadData.version}
             </caption>
           </div>
           <div className="mt-4">
             <div>Other downloads for Windows:</div>
```

Comment:
Unrelated nit: maybe this should be "Other Windows downloads:" to match other platforms?